karabo_data/tests/test_agipd_geometry.py

```diff
@@ -33,7 +33,7 @@ def test_write_read_crystfel_file(tmpdir):
     )
     np.testing.assert_allclose(loaded.modules[0][0].fs_vec, geom.modules[0][0].fs_vec)
 
-    # Load the geometry file with cfelpyutils and test the ridget groups
+    # Load the geometry file with cfelpyutils and test the rigid groups
     geom_dict = load_crystfel_geometry(path)
     quad_gr0 = ['p0a0', 'p0a1', 'p0a2', 'p0a3', 'p0a4', 'p0a5', 'p0a6', 'p0a7',
```

Comment:
just some synstastic sugar and I think I did this in the first place but maybe, if you want, you'll could use itertool's product to shorten this a bit ?

Reply:
Thanks, this is a good idea. Done.